infra/op-replica/kustomize/overlays/mainnet-replica-0-5-14-beta/patches/replica-healthcheck.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: replica-healthcheck
+spec:
+  template:
+    spec:
+      containers:
+        - name: replica-healthcheck
+          env:
+            - name: REPLICA_HEALTHCHECK__ETH_NETWORK_RPC_PROVIDER
```

Comment:
Shouldn't this be `REPLICA_HEALTHCHECK_REFERENCE_RPC_PROVIDER`? Newer versions of the replica-healthcheck use a different env name.

Reply:
These replicas still run both healthchecks because I'm paranoid.

```
resources:
...
  - ../../bases/replica-healthcheck
  - ../../bases/replica-healthcheck-v1
  ```

This patch is for the < v1 version.

For the v1 replica-healthcheck, we just include an env file

https://github.com/ethereum-optimism/optimism/pull/2715/files#diff-51f795f160a64f70ef947fef6ee9e1408d176e3255397bdda4ec81d4f60917fc